### ISSUE TITLE
Dependency versions

### ISSUE THREAD

**Issue by DStickley** [CaitinZhao/cvpr2019_Pyramid-Feature-Attention-Network-for-Saliency-detection#19]
Hi, could you please share the versions of the dependencies you used:
 
  Tensorflow (-gpu)
  Keras
  numpy
  opencv-python
  matplotlib

I'm having some issues running the notebook test that I think are due to some version inconsistencies.

**Comment by meghanakanapaneni** [comment]
Same issue. Please provide the versions of the dependencies.

**Comment by dongchankim-samsung** [comment]
I successfully ran demo.ipynb with following version and modification
docker base: Tensorflow 1.15.2-gpu-py3
keras: 2.2.4
modify xrange function to range
modify all C/2, C/4 operations in attention.py to int(C/2), int(C/4)
Hope this will help.

**Comment by dckim-o** [comment]
I successfully ran demo.ipynb with following version and modification
docker base: Tensorflow 1.15.2-gpu-py3
keras==2.2.4
numpy==1.19.1
opencv-python==4.2.0.32
matplotlib==3.1.3
modify xrange function to range
modify all C/2, C/4 operations in attention.py to int(C/2), int(C/4)
Hope this will help.

**Comment by DStickley** [comment]
Ah great thanks, I got it working too, although I did have to change conv_utils.normalize_data_format(data_format) to keras.backend.common.normalize_data_format(data_format) in bilinear_upsampling.py

this was with the same dependencies but numpy==1.18.1
